### PR TITLE
docs: add story for < Link /> and < StoryDescription />

### LINKED ISSUE #107: docs: Create basic story for < Link />, < StoryDescription />
[Monday ticket](https://monday.monday.com/boards/3532714909/pulses/5372933300)

Add basic story by structure similar to existing ones

https://github.com/mondaycom/vibe-storybook-components/blob/master/COMPONENTS_DOCUMENTATION_GUIDELINES.md

### PR DESCRIPTION
**Pull request by Dhoni77**
Closes #107

- Added stories for `<Link/>` and `< StoryDescription />` components

### CODE REVIEW

File: src/components/story-description/__stories__/story-description.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+};
+export default meta;
+
+type Story = StoryObj<typeof StoryDescription>;
+
+export const Overview: Story = {
+  args: {
+    description: 'Deprecated',
+    children: <StatusTag type="deprecated" />,
+    justify: 'Center',
```
Comment (SergeyRoyt) [inline src/components/story-description/__stories__/story-description.stories.tsx:17]:
```suggestion
    justify: FlexJustify.CENTER,
```
This should probably fix build issue

File: src/components/link/__stories__/link.stories.tsx
Comment (SergeyRoyt) [inline src/components/link/__stories__/link.stories.tsx:1]: Not sure why it occurs on some files and not on others, but, please, fix your IDE config to save line endings in Unix variant 🙏
Worst case you can just run `npm run prettier:fix`, it also should fix this

**Review by SergeyRoyt — CHANGES_REQUESTED** [review]
Great! 👍
Let's just make the checks green

File: src/components/link/__stories__/link.stories.tsx
Comment (Dhoni77) [reply src/components/link/__stories__/link.stories.tsx:1]: I ran npm run prettier:fix before commiting my code.

**Review by SergeyRoyt — APPROVED** [review]
Great stuff! Thanks a lot! 😃

**Pull request merged**